ghcide/src/Development/IDE/Plugin/HLS/GhcIde.hs

```diff
@@ -23,6 +23,7 @@ data Log
   = LogNotifications Notifications.Log
   | LogCompletions Completions.Log
   | LogTypeLenses TypeLenses.Log
+  | LogHover Hover.Log
```

Comment:
From tests: this case is missing in `instance Pretty Log` below.

src/Development/IDE/Plugin/HLS/GhcIde.hs:(30,12)-(33,38): Non-exhaustive patterns in \\case\n", _xdata = Nothing}